dist/origin-web-common.js

```diff
@@ -896,44 +896,6 @@ angular.module('openshiftCommonUI')
       }
     };
   }]);
```

Comment:
from the perspective of what should happen in the console, we never want OPENSHIFT_CONFIG being included in dist, that file should always be guaranteed to come from the generated config.js and nowhere else.  So removing it from common's dist is correct.  If that breaks the catalog app for some reason then we'll need to fix the catalog app.